dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js

```diff
@@ -5,13 +5,19 @@ import Details from './Details'
 import Boost from './Boost'
 import Availability from './Availability'
 import Review from './Review'
+import withWallet from '../../../../hoc/withWallet'
 
 const FractionalHourlyListing = props => (
   <Steps
     {...props}
     steps={[
       { step: 1, component: Details, require: 'subCategory' },
-      { step: 2, component: Boost, path: 'boost', require: 'title' },
+      {
+        step: 2,
+        component: withWallet(Boost),
```

Comment:
I think we can wrap the Boost component itself before it's exported, similar to how we do it [here](https://github.com/OriginProtocol/origin/blob/master/dapps/marketplace/src/pages/messaging/Messages.js#L150)

Reply:
Done.